conans/test/functional/toolchains/cmake/cmakedeps/test_cmakedeps_components_names.py

```diff
@@ -899,4 +899,4 @@ def build(self):
     client.save({"conanfile.py": consumer, "CMakeLists.txt": cmakelists})
     client.run("create .")
 
-    assert "Target name 'mypkg::name' already exists." in client.out
+    assert "Target name 'mypkg::name' already exists." not in client.out
```

Comment:
I was doubting if keeping these tests or not... I think it could be helpful to have them. Wdyt?

Reply:
Negative "not in" checks are low value, because of false positives. The message can change to "hello::hello exists" and it would still be true. Not harm to leave them though, you can leave them, no problem.

Reply:
You're right. Then, let's get rid of it.